### ISSUE TITLE
Missing translations or string identifiers?

### ISSUE THREAD

**Issue by radinamatic** [learningequality/kolibri#7273]
### Observed behavior

Encountered while testing #7259: 

All the languages seem to be missing the translation for `CREATE AN ACCOUNT` (`AuthBase.createAccountAction`), and some even for `Sign into` (`SignInPage.signInToFacilityLabel`).

![LEDev20 04 (start)  Running  - Oracle VM VirtualBox_754](https://user-images.githubusercontent.com/1457929/86974453-a82c5200-c176-11ea-804e-812ff538be98.png)

![LEDev20 04 (start)  Running  - Oracle VM VirtualBox_755](https://user-images.githubusercontent.com/1457929/86974476-afebf680-c176-11ea-8afa-c9077fdd62d9.png)

![LEDev20 04 (start)  Running  - Oracle VM VirtualBox_756](https://user-images.githubusercontent.com/1457929/86974486-b7130480-c176-11ea-9732-9d3bbc3f6a78.png)

`CREATE AN ACCOUNT` string has been around for several releases now (so it's definitely in the translation memory and can be re-used), but the identifier must have changed. No clue why it is not on Crowdin already, though... :confused: 

There has been a last-minute original string change from `Sign into` to `Sign in to`, and that might be the reason why is it missing on `sw-tz`, `bg-bg`, `my`, `bg-bg`, `ar`, `ur-pk`, `gu-in`, `ko`. 

cc @indirectlylit 

### Expected behavior
All strings translated and displaying correctly in supported languages.

### User-facing consequences
Impossible to understand important strings on the sign in page.

### Errors and logs
```
vue-intl.dev.js:3158 [Vue Intl] Missing message: "SignInPage.signInToFacilityLabel" for locale: "sw-tz", using default message as fallback.

vue-intl.dev.js:3158 [Vue Intl] Missing message: "AuthBase.createAccountAction" for locale: "sw-tz", using default message as fallback.
```

### Steps to reproduce
…

### Context
 * Kolibri version: current `release-v0.14.x` branch
 * Operating system: Ubuntu 18.04
 * Browser: all

**Comment by nucleogenesis** [comment]
Just a quick note:

`SignInPage.signInToFacilityLabel` is "Sign into" - not "Sign in to" - if this was intended to be a change included in 0.14 then I think we'll have to make an issue about it for the next round of translations.

PR incoming with fix

**Comment by radinamatic** [comment]
@nucleogenesis The change has been already committed in `develop` by @indirectlylit, I thought it was in the 0.14 :confused: 

https://github.com/learningequality/kolibri/commit/51c144a50b3bf2e39b8491eed113f44b719e561d#diff-599bf161f5d88e89f1ca26d00c7285bc
